### ISSUE TITLE
ToggleBreak adds actions to history?

### ISSUE THREAD

**Issue by kmindi** [ibogicevic/timeslice-chronometer#37]
Is this expected behavior, that when I pause the video, an additional history item is created with the action key of the current active key/slice?

**Comment by ibogicevic** [comment]
No, that is not intended. Pausing should neither increase the slice count nor be recorded in the history. I agree that this is a bug.
